_posts/0001-01-01-faq.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
I made this structural change as you suggested as well.  You make good points about the pages vs blog posts semantically speaking and it was pretty easy to make this change.  It actually simplified the logic to display the menu items, so that code is now firmed up a bit too.